src/common_swagger_api/schema/badges.clj

```diff
@@ -0,0 +1,29 @@
+(ns common-swagger-api.schema.badges
+  (:use [common-swagger-api.schema :only [describe]])
+  (:require [schema.core :as s])
+  (:import [java.util UUID]))
+
+(def SubmissionField (describe String "The JSON-encoded submission"))
+
+(s/defschema Submission
+  {:id
+   (describe UUID "The UUID for this submission")
+
+   :submission
+   SubmissionField})
```

Comment:
If you wanted to add some extra validation for the submission field, you can use the `AnalysisSubmission` schema defined in the `apps` service: https://github.com/cyverse-de/apps/blob/master/src/apps/routes/schemas/analysis.clj#L62-L115.

Reply:
This should be done now. Added the schema for AnalysisSubmission to common-swagger-api.schema.apps and used it here.